### PR TITLE
697 - Returning to submit page at a later date changes the date to the previous date

### PR DESCRIPTION
**Pull request by marcellmueller**
This bug was from when we moved the form_data to it's own table. Just had to use the `updated_at` column from `form_data` instead of the `application` table one.

### CODE REVIEW

File: app/components/Form/ApplicationForm.tsx [hunk 1]
```diff
@@ -275,6 +274,8 @@ const ApplicationForm: React.FC<Props> = ({
       newFormData[sectionName] = { ...calculatedSectionData };
     }
 
+    console.log(updatedAt, new Date(updatedAt));
+
```
Comment (matthieu-foucault) [inline app/components/Form/ApplicationForm.tsx:278]:
```suggestion
```

**Review by matthieu-foucault — COMMENTED** [review]
Do we want a test with this bugfix, to make sure the bug doesn't come back next time there's a refactor?

**Comment by matthieu-foucault** [comment]
@marcellmueller , actually looks like this was tested, and the test data needs updating

**Comment by marcellmueller** [comment]
> Do we want a test with this bugfix, to make sure the bug doesn't come back next time there's a refactor?

Yeah I thought about that, thanks for pointing out the current tests. I just had to update all the payloads and move the updatedAt fields to the formData objects.

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**